RELEASE-NOTES.txt

```diff
@@ -5,10 +5,10 @@
 1.24.0
 ------
 * New block: Latest Posts
-* [Android] Can now scroll post when in landscape orientation with the soft keyboard displayed
```

Comment:
This shouldn't be part of release notes as it was removed: https://github.com/wordpress-mobile/gutenberg-mobile/pull/1989/files#diff-a3d054590c32de57d87be233927e755dL3
But for some reason, it was reverted from revert.